lib/maze/macos_utils.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Maze
+  class MacosUtils
+    class << self
+      def capture_screen(scenario)
+        path = File.join(File.join(Dir.pwd, 'maze_output'), 'failed', scenario.name.gsub(/[:"& ]/, "_").gsub(/_+/, "_"))
```

Comment:
`scenario.name.gsub(/[:"& ]/, "_").gsub(/_+/, "_")` now appears in the codebase 3 times.  I'd create a new method in `Maze::Helper` along the lines of 
```
def to_friendly_filename(string)
  string.gsub(/[:"& ]/, "_").gsub(/_+/, "_")
end
```